src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
So `IndexRef ` contains information on the index ordering that can be used by your new operator, but I don't see where that's persisted in the catalog metadata. And the only existing implementation `createIndexIfNotExistPostgres` apparently ignores that information (but I guess that's something you can fix)!